### DISCUSSION TITLE
My incoming header showing this message

### DISCUSSION THREAD

**Discussion by RAKHAN1983** [postalserver/postal#2139] [Help with using Postal]
Attached is the screenshot of the incoming message i am receiving. 
Email is correct. I have verified
By looking the error can someone confirm that is this email is added in UNSUB Folder? Please confirm where i can get the list of unsub email list?
![Untitled](https://user-images.githubusercontent.com/112633343/187921691-fe16129b-e50b-4637-a3d2-610637a1dacc.png)

**Comment by willpower232** [comment]
Postal has no mailing list features, you need to implement that yourself with something else and then use Postal if you want to send emails through it for whatever reason.

https://docs.postalserver.io/welcome/faqs#can-you-add-mailing-list-feature
